tests/pre-command.bats

```diff
@@ -11,7 +11,7 @@ load "$BATS_PATH/load.bash"
 
   stub aws "sts assume-role --role-arn role123 --role-session-name aws-assume-role-buildkite-plugin-42 --duration-seconds 3600 --query Credentials : cat tests/sts.json"
 
-  run $PWD/hooks/pre-command
+  run $PWD/hooks/environment
```

Comment:
I think `tests/pre-command.bats` should be deleted, since it's essentially renamed to `tests/environment.bats`?

Reply:
I swear I did that, I wonder what went wrong.

Reply:
Ok, ⚡️